xlators/storage/posix/src/posix-gfid-path.c

```diff
@@ -117,7 +118,18 @@ posix_get_gfid2path(xlator_t *this, inode_t *inode, const char *real_path,
             if (size == 0)
                 goto done;
         }
-        list = alloca(size);
+
+        /* iot_worker stack size is 256k, in case of a file having huge number
+           of backend xattrs size could be greater than 256k and iot_worker thread
+           is crashed so to avoid a crash use CALLOC if size is greater than 32k
+        */
+        if (size > 32768) {
+            list = GF_CALLOC(1, size, gf_posix_mt_char);
```

Comment:
nit: I think you can GF_MALLOC(), since we memcpy the data into it or sys_llistxattr() fills it.